### ISSUE TITLE
Images as links

### ISSUE THREAD

**Issue by brian-rose** [jupyter-book/mystmd#1695]
## Proposal

Images, particularly logo and thumbnail images, are often clickable links on web pages. Currently this is possible in myst using plain markdown, e.g. `[![]()]()`. But then we don't have access to the configuration options offered by the `{image}` directive.

So my proposal is basically to add this functionality, perhaps with a new `:url:` argument to the image directive.

## Additional notes

The specific use case I'm thinking of here is for logos, which naturally want to link back to landing pages for whatever the logo is representing. The native logo images are often large, high-res image files and we want to scale them down for display, e.g. using the `:width:` option.

The need to scale down an image is one of the main reasons I reach for the `{image}` directive over the native Markdown image.

**Comment by choldgraf** [comment]
Agreed this would be useful - as a reference, the way that Jupyter Book currently handles this is with a `:target:` parameter:

````
```{image} foo.png
:target: https://google.com OR `myst-ref`
```
````

**Comment by mforbes** [comment]
Is there any workaround in the meantime beyond dropping to HTML?
